### DISCUSSION TITLE
PV Überschuss

### DISCUSSION THREAD

**Discussion by maggo1404** [cyberthom42/vaillant-arotherm-plus#24] [Q&A]
Hi,

Mit SMA-Homemanger und einem Vailant internetmodul (zb vr921), soll über eebus (2x e) der PV überschuss besser genutzt werden. Gibt es schon einen Weg die überschuss-daten mit ebusd in die anlage zu bekommenn. 

Hintergrund ist, das damit eine bessere Verbrauch als nur vollgas über den SG-Schaltkontakt möglich ist.

**Comment by cluk1** [comment]
Hi, 
ich habe hier eine Warmwasser- und eine Gebäude-Überheizung mit PV Überschuss implementiert ohne SG ready. Du findest die Automatisierungen in https://github.com/cluk1/homeassistant-config/ im Verzeichnis packages/photovoltaik/
Lass mich gerne wissen, ob dir das hilft.
Danke, Christoph

**Reply by maggo1404** [reply]
Danke für das Repo, ich verstehe es leider nicht was die Regelung macht?

Ich bin mit den Automationsscripte nicht so gut vertraut.

**Reply by chrizzzp** [reply]
@cluk1 
Danke, sehr interessant!

Kannst Du noch Deine CSVs hinterlegen oder kurz erklären, welche ebusd-Entitäten Du in den PV-Automatisierungen verwendest? Wenn ich das richtig interpretiert habe, erhöhst Du z.B. die Temperatur einer Helferentität auf 22°C (`input_number.flat_temperature_setpoint`), um bei PV-Überschuss das Haus etwas zu überheizen. Welche ebusd-Entität wird letztendlich verstellt (Wunschtemperatur?)?

Und bei der WW-Aktivierung wird ein Skript `script.warmwater_settings` ausgeführt. Was macht das genau, d.h. wie wird die WW-Aufladung getriggert?

**Reply by cluk1** [reply]
@chrizzzp gern:
CSVs: die sind hier: https://github.com/cluk1/ebusd-configuration

Gebäudeüberheitzung: Die Automatisierung packages/heating/heating_automation_power_auto.yaml regelt ebusd/ctlv2/z1HeatingRoomTempDesiredManualControlled (Zonen Wunschtemperatur) so, dass nach Möglichkeit in allen Wohnungen input_number.flat_temperature_setpoint als Temperatur erreicht wird.
Zur Gebäude Überheizung wird dann einfach input_number.flat_temperature_setpoint von 20 auf 22 Grad angehoben und die obige Automatisierung einmal angestoßen.

Warmwasserüberheizung:
Das Script ist im repo in scripts.yaml und setzt den Warmwasser Modus der WP (Eco / Normal), den Setpoint, die Maxchargetime, Chargehysteresis und den Operationmode (auto/heat) je nach übergebeen Parametern. 
Mit kleiner Hysterese (5), Setpoint (65), Maxchargetime (120), opmode (heat) und hwcmode(normal), geht die WP dann unmittelbar in den Warmwassermodus.

**Reply by chrizzzp** [reply]
@cluk1 

Danke für die Erklärungen!

> Mit kleiner Hysterese (5), Setpoint (65), Maxchargetime (120), opmode (heat) und hwcmode(normal), geht die WP dann unmittelbar in den Warmwassermodus.

Den Modus 'heat' gibt es in meinen Templates nicht für den HwcOpmode (Template opmode2):
`opmode2,UIN,0=off;1=time controlled;2=manual`

Entspricht 'heat' dem opmode 2 (manual), der im Regler (VRC720) auch 'manuell' heißt, sprich WW-Bereitung ist nicht zeitgesteuert, sondern permanent an (wenn Bedingungen für WW-Anforderung erfüllt)? Wie verhält sich der Opmode 'auto', ist das zeitgesteuert?

**Reply by cluk1** [reply]
Ja, genau. opmode heat ist WW permanent an (und Zirkulationspumpe übrigens auch). Ich habe für Heizung und Warmwasser ein climate device definiert: packages/hotwater/hotwater_device.yaml um das einfacher auch über die UI steuern zu können. Dort siehst du auch die genauen ebus parameter und das Mapping der Werte. 
Das Heizungs Device ist in packages/heating/heating_device.yaml

**Reply by mf0o** [reply]
Guten Morgen Christoph.
Tolles Repository und super Dokumentation, vielen Dank!
Das wird für mich der Startschuss um sich Homeassistant mal tiefer anzusehen und ggf. einen Schwenk von FHEM einzuleiten.

**Comment by chrizzzp** [comment]
Finde es gut, daß das Thema PV-Überschuss-Nutzung über ebusd mal hochgebracht wird. Ich denke es gibt hier schon einige gute Ansätze, die man mal zusammentragen könnte, um vielleicht einen Überblick in Form einer  einen Wiki-Seite (@cyberthom42 ) zu erstellen?

Sicherlich hängen die Ansätze auch von einzelen WP-Setups ab, aber was ich bisher so gehen (gelesen) habe wie man PV-Überschuss nutzen kann umfasst z.B.

- Überheizung des Hauses durch Anheben der Heizkreis-Mindestvorlauftemperatur oder Zonen-Wunschtemperatur 
- (Warmwasser)-Speicher aufladen über PV/SG-Kontakte (max. Offset +15 K)
- Warmwasser-Speicher aufladen über Sonderfunktion "Warmwasser schnell" (HwcSFMode 6)
- Warmwasser-Speicher aufladen über WW-Betriebsmodus "manuell" (opmode 'heat'/'manual') und entsprechenden WW-Parametern
- Nutzung externer (virtueller) Heizkreise zur Generierung von Wärmeanforderungen (z.B. externer WW-Kreis, erfordert aber Funktionsmodule (z.B. VR71/FM)
- Modulation der WP-Leistung über Flüsterbetrieb (an/aus, Leistungsdrosselung im Flüsterbetrieb) 
- Modulation der WP-Leistung durch adaptives Verstellen der Mindestvorlauftemperatur oder HcXExcessTemp (Überheizung Heizkreisvorlauf)
- optionale Zuschaltung vom Heizstab (z.B. über Alternativpunkt), über Heizstableistungsbegrenzung gut modulierbar

Auch die Erfassung/Vorhersage des PV-Überschusses könnte man mal zusammenfassen:

- welche PV-Vorhersagetools sind nutzbar/sinnvoll? (z.B. Solcast, Forecast.Solar, solarprognose.de)
- wie kann man Takten bei kurzzeitigen PV-Spitzen vermeiden (gleitende Mittelwerte, Energieintegral "fixieren")
- was sind gute Zuschalt-/Wegschaltbedingungen (die man in Automatisierungen realisieren kann)?

Edit: ich aktualisiere diesen Beitrag immer mal als "Ideensammlung"...

Ich selbst beschäftige mich auch schon eine Weile mit dem Thema. Da ich vermutlich ein relativ exotisches Setup habe (Kombination von überdimensionierter Wärmepumpe und wasserführendem Kamin, die alternativ einen Heizungspufferspeicher beladen) ist es vielleicht nicht unbedingt von allgemeinem Interesse, aber auch eine weitere diskutable Variante, denke ich. Da ich keinen Batteriespeicher habe, nutze ich z.B. die Aktivierung zusätzlicher (virtueller) Heizkreise, um meinen Heizungspufferspeicher mit PV-Überschuss aufzuheizen (die Aktivierung des Heizkreises erzeugt eine direkte Wärmeanforderung). Das ganze mache ich übrigens auch über die Automatisierungen von Homeassistant und das ebusd-Addon.

Über den tatsächlichen Nutzen/die Effizienz der PV-Überschussnutzung während der Heizperiode kann man sich sicher streiten, aber ich denke das Thema wird auch zusätzlich interessant werden, sobald die flexiblen Stromtarife flächendeckend verfügbar und attraktiver werden.

**Reply by cyberthom42** [reply]
Das sind gute Ansätze. Eins sollte man dabei beachten: Wenn man das rein nach PV-Überschuss macht versaut man sich dadurch die Effizienz, gerade an Tagen mit viel Wechsel von Sonne zu Schatten. Wenn man z.B. die Heiztemperatur bei Sonne anhebt und dann durch ein Wolke wieder absenkt springt das Energieintegral nach oben und führt ggf. zu einer Abschaltung. Dadurch hat man sich dann die Effizienz versaut. Ich hatte das dann so implementiert, dass ich das Energieintegral immer nachgeregelt habe, sodass die WP immer bei 25% bleibt. 

Bei Warmwasser muss man beachten, dass Warmwasser durchläuft wenn es einmal gestartet ist. Es sei denn man senkt die WUnschtemperatur danach wieder ab. Macht aber wenig Sinn. Ich lasse Warmwasser z.B. immer Mittags machen. Besser wäre, Warmwasser nach Solarprognose zu steuern. 
Den Heizstab kann man nicht im Heizbetrieb dazuschalten (siehe dazu andere Threads hier). Man kann den Heizstab aber bei WW zuschalten und hier auch recht schnell die Leistung hoch und runter drehen. Abschalten lässt er sich dann im aktuellen Warmwasser-Lauf nicht mehr, man kann die Leistung aber auf 0 setzen

Zusammen gefasst sehe ich gutes Potential bei einer Prognose-basierten Steuerung. 

Zu den Fragen:

- Bei den Prognose-Tools bin ich bei solcast.solar hängen geblieben. Das hatte über die Dauer gesehen die beste Prognose-Qualität. 
- Takten kann man u.a. vermeiden in dem man das Energieiintegral im Auge behält. Dazu hatte ich mehrere Strategien. Die einfachste Strategie war kurzzeitig das Energieintegral zu senken in dem ich für 30 Sekunden die MinSollTemp auf den maximalen Wert gehoben habe. Das kann man mehrmals hintereinander machen. Eine andere Strategie ist es die MinSollTemp immer anzuheben/senken je nachdem wie hoch das Energieintegral ist
- Der Flüsterbetrieb ist bei mir eigentlich immer an bis in den tiefen Winter wenn wir dann permanent Minus-Grade haben

**Reply by chrizzzp** [reply]
> Das sind gute Ansätze. Eins sollte man dabei beachten: Wenn man das rein nach PV-Überschuss macht versaut man sich dadurch die Effizienz, gerade an Tagen mit viel Wechsel von Sonne zu Schatten. Wenn man z.B. die Heiztemperatur bei Sonne anhebt und dann durch ein Wolke wieder absenkt springt das Energieintegral nach oben und führt ggf. zu einer Abschaltung. Dadurch hat man sich dann die Effizienz versaut. Ich hatte das dann so implementiert, dass ich das Energieintegral immer nachgeregelt habe, sodass die WP immer bei 25% bleibt.

Hier wäre es m.E. schön mal konkret zusammenzutragen, wie Du z.B. das Energieintegral nachgeregelt hast. Meines Erachtens geht das ja nicht direkt, sondern nur über die Anpassung von Soll-Vorlauftemperaturen im Verhältnis zum Vorlauf-Ist (OK, sehe gerade, daß hast Du weiter unten geschrieben). Aber es wäre auch interessant zu wissen, in welchen Bereichen man das Energieintegral schieben muß, um damit die Leistung z.B. bei 25% Leistung zu halten.

Was (kurzfristige) Verschattungen angeht, kann man ja mit gemittelten PV-Überschussmessungen arbeiten, um ein frühzeitiges Abschalten zu verhindern. Wenn es verschattet bleibt, muß man eben wegschalten. Finde ich nicht so dramatisch, solange man nicht 5 min später wieder zuschalten will.
Und was die Effizienz angeht:  hier wäre ich bereit Einbussen hinzunehmen, wenn man bedenkt, daß man (zumindest zeitweise) für lau (PV-Überschuss) heizen kann. Spiegelt sich ja erstmal auch nicht in den (eingebauten) Effizienzwerten wieder. Und selbst die Wärme, die man z.B. durch Verschattung mit Netzstrom "zu viel" reingepumpt hat, ist ja nicht ganz verloren, sondern verzögert das nächste Nachheizen.

> Bei Warmwasser muss man beachten, dass Warmwasser durchläuft wenn es einmal gestartet ist. Es sei denn man senkt die WUnschtemperatur danach wieder ab. Macht aber wenig Sinn. Ich lasse Warmwasser z.B. immer Mittags machen. Besser wäre, Warmwasser nach Solarprognose zu steuern. 

Bist Du sicher? Für welchen WW-Betriebsmodus (HwcOpmode) gilt das? Ich kann mir nicht vorstellen, daß der WW-Kreis weiterläuft, wenn man den auf 'aus' schaltet (das kann man ja per ebus leicht machen, um ihn später wieder z.B. auf "zeitgesteuert" oder "manuell" zu stellen).

Im externen Warmwasserkreis geht das übrigens problemlos (so betreibe ich es). Ich setzte auch die WW-Solltemperatur und den ExtHwcOpmode auf 'manuell' und zum (vorzeitigen) Wegschalten auf 'aus'.

> Den Heizstab kann man nicht im Heizbetrieb dazuschalten (siehe dazu andere Threads hier). Man kann den Heizstab aber bei WW zuschalten und hier auch recht schnell die Leistung hoch und runter drehen. Abschalten lässt er sich dann im aktuellen Warmwasser-Lauf nicht mehr, man kann die Leistung aber auf 0 setzen

Da habe ich andere Erfahrungen: Bei mir ist der Heizstab in der Hydraulikstation und ich kann ihn jederzeit durch Setzen des Alternativpunkts für's Heizen (z.B. auf 40°C) exklusiv für den Heizbetrieb aktivieren, sobald ich eine Wärmeanforderung generiere (z.B. Vorlauf-Sollwerterhöhung, Energieintegral bleibt aber stehen, das gilt nur für den Kompressor). Wegschalten geht auch, wenn die Wärmeanforderung weg ist (wenn auch mit kleiner Verzögerung, die ich noch nicht verstehe). Und die Leistungsregelung geht ja über die Maximalleistung des Heizstabs (bei mir 1-9 kW). Natürlich alles (auch) per Ebus.

Im Gegensatz dazu habe ich noch keinen Alternativpunkt für den WW-Betrieb gefunden. Hier gibt es im VRC720/2 nur den Bivalenzpunkt (ebus-Register bekannt), d.h. kein exklusiver Betrieb des Heizstabes. Im aktuellen VRC720/3 gibt es jetzt auch einen für den WW-Betrieb (wäre mal interessant das ebus-Register dafür zu erfahren...).

> Zusammen gefasst sehe ich gutes Potential bei einer Prognose-basierten Steuerung.
> 
> Zu den Fragen:
> 
>     * Bei den Prognose-Tools bin ich bei solcast.solar hängen geblieben. Das hatte über die Dauer gesehen die beste Prognose-Qualität.

Hier wäre es interessant zu wissen, welche Vorhersagewerte Du dann tatsächlich heranziehst. Nur den verbleibenden prognostizierten PV-Ertrag für den Tag oder auch z.B. den Zeitpunkt des maximalen vorhergesagten Ertrages. Das (und einiges mehr) liefern die Modelle ja alles.
Ich benutze zur Zeit zwei Modelle, deren Prognosen ich beide berücksichtige, um zu entscheiden, ob es sich an diesem Tag überhaupt noch lohnt, die WP nochmal bei momentanem PV-Überschuss zuzuschalten (restliche prognostizierter Überschuss für diesen Tag). Ich nehme dann immer die "optimistischere" Prognose, um zu entscheiden... (sicherer wäre es wahrscheinlich, die pessimistischere Variante zu berücksichtigen). Heute lagen übrigens beide etwas daneben... ;-))

**Reply by mf0o** [reply]
Wir haben auch eine überdimensionierte flexoCOMPACT mit 500L Heizungspufferspeicher am gemischten Heizkreis.
Aufs Dach passten "nur" 8.5kWp die ich aktuell mittels einem Script vertüddelt hab. 

Die Wechselrichterdaten hole ich von SolarEdge über die API von deren Server, die (ausgewählten) Vaillant Counter per HTTP/REST alle 5min (/data/${circuit}/${counter}?maxage=120&exact=true) und das ganze Schiebe ich zusätzlich in eine Datenbank für eine Visualisierung mit Grafana.

Habe erst im Sommer mit der Implementation angefangen und daher erstmal die Warmwasser Überheizung gemacht.
Bei ausreichend Einspeisung der PV über mindestens 15min, erhöhe ich HwcTempDesired und schalte den Shelly am SGReady Kontakt. 
Mit der Kombination konnte ich auch im Sommer eine sofortige Heizungsanforderung erzeugen.
("Warmwasser schnell" lässt sich bei mir auch über **ebusctl w -c ctlv2 HwcSFMode 6** starten aber das nutze ich an anderer Stelle).

Wenn die neue Warmwasser-Zieltemperatur erreicht ist, schraube ich HwcTempDesired wieder runter, lasse den SGReady Kontakt aber geschlossen. Bei Bedarf läd bzw. überlad er dann noch den Heizungspufferspeicher - dazu will ich zukünftig noch die Vorlauftemperatur oder Hc2ExcessTemp erhöhen.

Das ganze wird unterbrochen sobald die PV-Produktion einen, momentan recht niedrigen, Wert unterschreitet.
Funktioniert soweit aber ist Gefühlt nur aus "Spucke und Zahnstochern" gebastelt.


Bei der Solarvorhersage hatte ich damals Solarprognose.de und forcast.solar für meine Anlage mit eingebunden und etwas an den Parametern gespielt. Bisher habe ich dazu aber nur die Visualisierung zum Vergleich mit der Produktion gemacht und generell versucht jeglichen Heizbetrieb in nicht-Sonnen-Stunden zu vermeiden.
<img width="1184" alt="image" src="https://github.com/user-attachments/assets/28cd726a-e947-46f5-b130-6a05cfd5a937">

**Reply by chrizzzp** [reply]
@mf0o Interesant, dann haben wir recht ähnliche Systeme. Wenn Du mehrere Heizkreise hast, nehme ich an Du hast auch das VR70/VR71-Modul?

> Habe erst im Sommer mit der Implementation angefangen und daher erstmal die Warmwasser Überheizung gemacht. Bei ausreichend Einspeisung der PV über mindestens 15min, erhöhe ich HwcTempDesired und schalte den Shelly am SGReady Kontakt. Mit der Kombination konnte ich auch im Sommer eine sofortige Heizungsanforderung erzeugen. ("Warmwasser schnell" lässt sich bei mir auch über **ebusctl w -c ctlv2 HwcSFMode 6** starten aber das nutze ich an anderer Stelle).

Wenn Du sowieso `HwcTempDesired` bei PV-Überschuss per Ebus erhöhst, könntest Du auch gleich den HwcOpmode auf "manuell" (permanent an) stellen (`ebusctl w -c ctlv2 HwcOpMode 2`), dann müsste doch auch ohne Shelly/Schließen des SG-Kontaktes die WW-Aufladung starten, oder?

> Wenn die neue Warmwasser-Zieltemperatur erreicht ist, schraube ich HwcTempDesired wieder runter, lasse den SGReady Kontakt aber geschlossen. Bei Bedarf läd bzw. überlad er dann noch den Heizungspufferspeicher - dazu will ich zukünftig noch die Vorlauftemperatur oder Hc2ExcessTemp erhöhen.

Nutzt man den SGReady-Kontakt, kann man den Heizungsspeicher übrigens maximal 15K überheizen (maximaler Offset). Ob Hc2ExcessTemp auch einen maximalen Offset hat, weiß ich gerade nicht. Vielleicht kann man beide auch kombinieren?

Wenn man bei PV-Überschuss einen weiteren virtuellen Heizkreis (z.B. Hc3) als  Festwert-Heizkreis aktiviert, kann man den Sollwert frei wählen und auch höher überheizen. Dazu braucht man auch den SGReady-Kontakt nicht, aber das o.g. Funktionsmodul.

**Reply by mf0o** [reply]
Hi @chrizzzp. Genau, bei mir ist ein VR70 verbaut, der den Mischer zwischen Heizkreis und Heizungspufferspeicher steuert.

Das mit dem `HwcOpMode 2` kann ich gerne mal probieren.

Das mit dem virtuellen Heizkreis hatte ich mal probiert, aber das hat nicht wirklich gut geklappt bei mir (eventuell war meine Konfiguration fehlerhaft). Unterm Strich hat es mir nur die Taktung erhöht und nicht wirklich überheizt.

Interessanterweise komme ich mit dem `Hc2ExcessTemp` momentan recht gut hin. Anfangs hatte ich das Phänomen, dass er den Heizungspuffer nie wirklich getrennt aufgeladen hat und somit der Mischer meist einfach auf 100% lief.
Erst mit dem `Hc2ExcessTemp` (aktuell bei 5-7 K) kommt der Mischer so richtig im Einsatz. Mein Installateur hatte sich da schon die Zähne ausgebissen (obwohl Vaillant Platin Händler) genauso wie die Vaillant Hotline.
Lustigeweise finde ich den Parameter `Hc2ExcessTemp` im VR720 auch nicht - den konnten wir also bisher nie setzen.

Bei den aktuellen Temperaturen (3-10°C) habe ich zwischen 7-10 Taktungen pro Tag. 
Wie sieht es da bei euch aus?

**Reply by mf0o** [reply]
Achso, Heizbedarf entsteht immer dann, wenn meine Ist-Vorlauftemperatur  (VR70 Sensor S6) um 3K (ich glaube definiert in `CylinderChargeHyst`) niedriger ist als die Soll-Vorlauftemperatur. Dann wird der Puffer geladen bis die Puffer-Boden-Temperatur (VR70 Sensor S1) um `Hc2ExcessTemp` höher ist als Soll-Vorlauftemperatur.

Sieht dann ungefähr so aus:
<img width="977" alt="image" src="https://github.com/user-attachments/assets/711f0596-3dd2-40cd-a064-8f6aa526fd8a">

**Reply by chrizzzp** [reply]
@mf0o 

> Das mit dem virtuellen Heizkreis hatte ich mal probiert, aber das hat nicht wirklich gut geklappt bei mir (eventuell war meine Konfiguration fehlerhaft). Unterm Strich hat es mir nur die Taktung erhöht und nicht wirklich überheizt.

Die Frage wäre, was das Abschaltkriterium für den virtuellen Heizkreis war. Bei mir geht die WP aus, wenn das Energieintegral positiv wird. Hystereseeinstellung kann natürlich auch greifen oder ggf. ein Puffer-Temperaturfühler (gibt's bei mir nicht, da mein Pufferspeicher als Solarspeicher mit zwei separaten Fühlern konfiguriert ist).

> Interessanterweise komme ich mit dem `Hc2ExcessTemp` momentan recht gut hin. Anfangs hatte ich das Phänomen, dass er den Heizungspuffer nie wirklich getrennt aufgeladen hat und somit der Mischer meist einfach auf 100% lief. Erst mit dem `Hc2ExcessTemp` (aktuell bei 5-7 K) kommt der Mischer so richtig im Einsatz. Mein Installateur hatte sich da schon die Zähne ausgebissen (obwohl Vaillant Platin Händler) genauso wie die Vaillant Hotline. Lustigeweise finde ich den Parameter `Hc2ExcessTemp` im VR720 auch nicht - den konnten wir also bisher nie setzen.

`HcExcessTemp` erhöht m.E. direkt den Wert von VorlaufWPSoll (kann man aus der `SetMode`-Definition in der `hcmode.inc` auslesen). Mir ist auch nicht ganz klar, wann der Wert in der Vaillant-Regelung eine Rolle spielt. Ich vermute nur bei Direktkreisen, falls die Vorlauftemperatur des Heizkreises nicht erreicht wird. Beim gemischten Heizkreis erhöht es wie gesagt nur den Vorlauf-Sollwert der Wärmepumpe (ggf. Puffertemperatur), was der Mischer dann natürlich wieder zunichte macht (der Vorlauf-Sollwert des Heizkreises bleibt unverändert).

Interessant auch zu sehen, daß Deine Überheizung des Puffers ja nur recht kurz wirkt (30-60 min). Falls Fußbodenheizung, wieviel Quadratmeter werden denn beheizt und bei welcher Außentemperatur?

**Comment by chrizzzp** [comment]
Ein weiterer Ansatz:

Kennt eigentlich schon jemand das (ich glaube neue) Energiemanagement, daß Vaillant über die Plattform https://myvaillantweb.de/ anbietet? Ist wohl noch in der Entwicklung (beta), aber hier soll wohl eine (einsteigerfreundliche) Variante angbeoten werden, um abhängig von vorhergesagtem (!) PV-Überschuss die Wärmepumpe (ferngesteuert über das Internetgateway VR921) zu starten. Bisher kann man wohl nur den Warmwasserspeicher aufheizen lassen (nicht getestet), aber könnte mir vorstellen, daß sie hier vielleicht noch mehr Funktionen implementieren. Eigentlich keine schlechte Idee (bleibt natürlich weit hinter den Möglichkeiten einer lokalen Lösung zurück). Hat das schonmal jemand probiert?

**Reply by cyberthom42** [reply]
Ich habe mich noch nicht getraut, Vaillant zugriff auf mein VR921 zu gewähren was ja hierfür nötig wäre. Wenn die Zugriff haben sehen sie ja auch, dass dort der ebusd Controller drin hängt. Vaillant ist da glaube ich sehr allergisch dagegen, spätestens im Garantie-Fall.

**Reply by chrizzzp** [reply]
Gibt es da denn derartige Erfahrungen mit Vaillant? Allein die Präsenz eines ebus-Adapters muß ja nicht ursächlich mit einem Schadensfall zu tun haben. Oder habe ich da im Kleingedruckten was überlesen? ;-)
Aber klar ist: man sollte wissen, was man tut, sobald man mehr als nur Werte liest über den Ebus.

**Reply by maggo1404** [reply]
Die vailant Lösung ist halt ein prognostisches Verfahren. Bei Nebellagen kann man das komplett vergessen.

**Comment by cyberthom42** [comment]
> Hi,
> 
> Mit SMA-Homemanger und einem Vailant internetmodul (zb vr921), soll über eebus (2x e) der PV überschuss besser genutzt werden. Gibt es schon einen Weg die überschuss-daten mit ebusd in die anlage zu bekommenn.
> 
> Hintergrund ist, das damit eine bessere Verbrauch als nur vollgas über den SG-Schaltkontakt möglich ist.

EEBUS kann prinzipiell nicht mehr als ebus, da EEBUS ja nur die ebus-Befehle umsetzt. EEBUS ist von den Profilen sehr begrenzt. Da haben wir mit direktem ebus-Zugriff weitaus mehr Möglichkeiten

**Reply by maggo1404** [reply]
Das war der Hintergrund der Frage, das da schon die Register bekannt sind wo man beschreiben muss. Das die WP sinnvoll läuft.
